mocha/mocha.opts

```diff
@@ -0,0 +1,4 @@
+--recursive
+--webpack-config webpack.config.js
+--require mocha/setup.js
+--timeout 10s
```

Comment:
Well, it has a timeout of 1,5s so it's not surprising ;) The real solution is to get rid of the `setTimeout` call, not making sure tests can take longer.